homeassistant/components/lutron_caseta/light.py

```diff
@@ -92,6 +98,8 @@ def __init__(self, light: dict[str, Any], data: LutronCasetaData) -> None:
         self.supports_warm_cool = light_type in WARM_DEVICE_TYPES
         self.supports_warm_dim = light_type == DEVICE_TYPE_SPECTRUM_TUNE
         self.supports_spectrum_tune = light_type == DEVICE_TYPE_SPECTRUM_TUNE
+        self.supports_warm_dim = light_type == DEVICE_TYPE_COLOR_TUNE
```

Comment:
Won't these overwrite the lines above it? Ie if the light is spectrum tune the variables are set to true in the lines above this, then set back to false here because it is not a color_tune